docs/advanced/addons.md

```diff
@@ -16,4 +16,12 @@ v1.1.0 currently ships with two Addons:
 * [pcidevices-controller](./pcidevices.md)
 * [vm-import-controller](./vmimport.md)
 
-![](/img/v1.2/addons/DefaultAddons.png)
\ No newline at end of file
+![](/img/v1.2/addons/DefaultAddons.png)
+
+_Available as of v1.2.0_
+
+v1.2.0 ships two new Addons:
```

Comment:
```suggestion
v1.2.0 ships with two more Addons:
```

Reply:
`v1.2.0 ships two new Addons:` seems like v1.2.0 only has two addons, but it actually has four